src/dmd/link.d

```diff
@@ -220,9 +220,14 @@ public int runLINK()
             }
             for (size_t i = 0; i < global.params.libfiles.dim; i++)
             {
-                cmdbuf.writeByte(' ');
-                cmdbuf.writestring("/DEFAULTLIB:");
-                writeFilename(&cmdbuf, global.params.libfiles[i]);
+                // The `/DEFAULTLIB:` argument cannot be empty
+                const f = global.params.libfiles[i];
+                if (f && strlen(f) > 0)
```

Comment:
replace `strlen(f) > 0` with `f[0]`

Reply:
also, perhaps this check should be done where `libfiles[]` is filled, rather than where it is used.